libs/market-list/src/lib/components/select-market-columns.tsx

```diff
@@ -217,11 +231,11 @@ export const columns = (market: any, onSelect: (id: string) => void) => {
       onlyOnDetailed: false,
     },
     {
-      value: market.lastPrice ? (
+      value: market.data?.markPrice ? (
         <PriceCell
-          value={new BigNumber(market.lastPrice).toNumber()}
+          value={new BigNumber(market.data?.markPrice).toNumber()}
```

Comment:
question (non blocking): Why create big number here and then convert to number anyway? Will `Number(market.data?.markPrice ?? 0)` suffice?

Reply:
Same elsewhere in this file..